src/com/example/powerup/MainActivity.java

```diff
@@ -0,0 +1,45 @@
+package com.example.powerup;
+
+import com.example.powerup.R;
+
+import java.io.IOException;
+import java.util.List;
+
+import android.support.v7.app.ActionBarActivity;
+import android.app.ListActivity;
+import android.content.Intent;
+import android.database.sqlite.SQLiteDatabase;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.ArrayAdapter;
+
+public class MainActivity extends ListActivity {
+
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		Intent myIntent = new Intent(MainActivity.this, Game.class);
```

Comment:
Why are you doing this? Is it a temporal solution? You can specify your starting activity on your manifest and set it to Game directly.